client/src/components/Navbar.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import IconButton from "@mui/material/IconButton";
+import MenuRoundedIcon from "@mui/icons-material/MenuRounded";
+
+import "../css/Navbar.css";
+import { Link, NavLink } from "react-router-dom";
+
+export const Navbar = () => {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  function handleOpenNav(e) {
+    e.preventDefault();
+    document.getElementById("myNav").style.width = "100%";
+    console.log("The link has been  clicked.");
+  }
+
+  function handleCloseNav(e) {
+    e.preventDefault();
+    document.getElementById("myNav").style.width = "0%";
+    console.log("The link has been  clicked.");
+  }
+
+  return (
+    <nav>
+      <div id="myNav" class="overlay">
+        {/* <!-- Button to close the overlay navigation --> */}
+        <a href="javascript:void(0)" class="closebtn" onClick={handleCloseNav}>
+          &times;
+        </a>
+
+        {/* <!-- Overlay content --> */}
+        <div class="overlay-content">
+        <Link to="/">Home</Link>
+        <Link to="/AboutUs">About Us</Link>
+        <Link to="/Resources">Resources</Link>
```

Comment:
I thought we werent doing this anymore

Reply:
Do you mean the navbar itself? If so, I haven't heard about any plans for scrapping it so I'd assume not

Reply:
Sorry, referring to line 34 and 35; about us and resources pages

Reply:
deleted